### ISSUE TITLE
Dynamic sidebarMenu not working in 2.0.3

### ISSUE THREAD

**Issue by pisistrato** [RinteRface/bs4Dash#234]
Just bumped into this, the following code (from `?renderMenu`) generate and error
Warning: Error in renderUI: object 'expr' not found

```
library(shiny)
library(bs4Dash)

ui <- dashboardPage(
  dashboardHeader(title = "Dynamic sidebar"),
  dashboardSidebar(
    sidebarMenuOutput("menu")
  ),
  dashboardBody()
)

server <- function(input, output) {
  output$menu <- renderMenu({
    sidebarMenu(
      menuItem("Menu item", icon = icon("calendar"))
    )
  })
}

shinyApp(ui, server)
```

**Comment by DivadNojnarg** [comment]
In the meantime, try this, which contains the missing piece of code causing the issue:

```r
renderUser <- function(expr, env = parent.frame(), quoted = FALSE, outputArgs = list()) {
 if (!quoted) {
    expr <- substitute(expr)
    quoted <- TRUE
  }
  shiny::renderUI(expr, env = env, quoted = quoted, outputArgs = outputArgs)
}
```

**Comment by DivadNojnarg** [comment]
Fixed in CRAN 2.1.0
